### PR TITLE
Adds scikit-learn - yyyymm accuracy only

### PR DESCRIPTION
**Pull request by Zarkoix**
couldn't find exact day, so putting the 1st of the month

### CODE REVIEW

**Comment by jsrn** [comment]
This PR seems to remove the Keras entry. Could you please fix that? Thanks.

**Comment by jsrn** [comment]
Scratch that, I got you. 👌 Thank you for your contributions!
